modules/http/src/main/java/org/mule/module/http/internal/request/HttpClientFactory.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.http.internal.request;
+
+import org.mule.module.http.internal.request.grizzly.GrizzlyHttpClientConfiguration;
+
+/**
+ * Factory object for {@link HttpClient} mule-to-grizzly adapters.
+ */
+public interface HttpClientFactory
```

Comment:
The name of this interface doesn't mention grizzly and neither is it in the grizzly package, yet it receives a GrizzlyHttpClientConfiguration..